src/pages/Questionnaire.tsx

```diff
@@ -92,13 +96,33 @@ export function Questionnaire() {
   }));
 
   useEffect(() => {
```

Comment:
would be nice to add comments on top of useEffects so the next developer that works on your code knows quickly what is going on. So, for this one, you can say:

// Sets the `currentQuestion` state based on the value in local storage